cfme/services/catalogs/catalog.py

```diff
@@ -98,6 +99,7 @@ def update(self, updates):
         else:
             view.cancel_button.click()
         view = self.create_view(DetailsCatalogView, override=updates)
+        view.wait_displayed()
```

Comment:
Lines 102 and 103 can be combined into a single line:
``` assert view.wait_displayed()```

`wait_displayed()` returns the boolean of `wait_for`

Reply:
@sshveta I jumped the gun here, looks like that change hasn't been released yet :) @mshriver will be doing it after lunch

Reply:
OO I made the changes :)

Reply:
@jawatts : Failures are in test_access_control.py  since all the tests ran . Not related to changes made by me.

Reply:
wt.core 0.30.3 included with #8055, I've restarted PRT here as now wait_displayed should be returning boolean.